vm-ubuntu-20.04/user-common-bootstrap.sh

```diff
@@ -7,7 +7,7 @@ set -xe
 git clone https://github.com/mininet/mininet mininet
```

Comment:
@jafingerhut would it be possible to clone a specific tag from the mininet repository?
For example:
```
MININET_COMMIT="2.3.0"
git clone -b ${MININET_COMMIT} https://github.com/mininet/mininet mininet
```

Reply:
Definitely possible.  At this time, there does not appear to be a Mininet tag version that corresponds to the 2022-Apr-01 change made to the util/install.sh file in the Mininet repo.  I'm happy to leave this PR open until there is such a tagged version of Mininet, and when there is, add such a tagged version to this PR before it is merged in.

In the mean time, I can create a separate PR that uses a tagged version with the current Mininet patch.

Reply:
The most recent tag for Mininet appears to be over 1 year old, so I added another commit to this PR that uses a particular recent commit of Mininet.  See what you think.

Reply:
It looks good. Thank you for fixing this.